tests/tracking/fluent/test_fluent_autolog.py

```diff
@@ -218,10 +218,16 @@ def test_autolog_obeys_disabled():
 
     mlflow.autolog(disable=False)
     mlflow.utils.import_hooks.notify_module_loaded(sklearn)
-    assert not get_autologging_config("sklearn", "disable", False)
+    assert not get_autologging_config("sklearn", "disable")
     mlflow.sklearn.autolog(disable=True)
     assert get_autologging_config("sklearn", "disable")
 
+    mlflow.autolog(disable_for_unsupported_versions=False)
```

Comment:
Can we also test the case where we explicitly invoke `mlflow.autolog(disable_for_unsupported_versions=True)`?